### PR TITLE
fixes to a11y errors

### PR DESCRIPTION
**Pull request by sendilkumarn**
This fixes 2 a11y errors.

### CODE REVIEW

File: src/demo-app/index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!doctype html>
 {{!-- This file is an handlebar file that gets filled at build time. --}}
-<html>
+<html lang="en">
```
Comment (DevVersion) [inline src/demo-app/index.html]: You have to apply this to the `e2e-app` as well.

Comment (sendilkumarn) [inline src/demo-app/index.html]: Will update this

File: src/demo-app/button/button-demo.html [hunk 2]
```diff
@@ -11,8 +11,8 @@
   </section>
 
   <section>
-    <a href="http://www.google.com" md-button color="primary">link</a>
-    <a href="http://www.google.com" md-raised-button>link</a>
+    <a href="http://www.google.com" md-button color="primary">Google</a>
```
Comment (jelbourn) [inline src/demo-app/button/button-demo.html]: The text should be `"SEARCH"`

File: src/demo-app/index.html (hunk 1 above)
Comment (jelbourn) [inline src/demo-app/index.html]: @robertmesserle is doing this in #650

**Comment by jelbourn** [comment]
Closed in favor of #741

**Pull request closed without merging**